sdk/core/core-util/src/bytesEncoding.browser.ts

```diff
@@ -0,0 +1,95 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+declare global {
+  // stub these out for the browser
+  function btoa(input: string): string;
+  function atob(input: string): string;
+}
+
+export type EncodingType = "utf-8" | "base64" | "base64url";
+
+/**
+ *
```

Comment:
missing a description sentence for the doc string here and below, this is why api-extractor is saying "undocumented"

Reply:
Should have been resolved.